### DISCUSSION TITLE
Hubs Blender Addon UV-Scroll won't work in Spoke/Hubs

### DISCUSSION THREAD

**Discussion by carlagnyu** [mozilla/hubs#3321] [General Questions & Discussion] [answered]
The hubs blender addon will not  apply UV-Scroll to my textures. I've tried it 15 times, with different textures, different versions of Blender, different objects. Nothing works!! When I bring it into Spoke and check the gltf properties it always gives me the warning that i have an UNSUPPORTED_EXTENSION. What what am I doing wrong? 
![image](https://user-images.githubusercontent.com/55764072/98907226-7e6c9a00-248c-11eb-8e81-036aab1a3b2d.png)

**Comment by matthewbcool** [comment, marked as answer]
The gltf validator seems to always give us the warning of UNSUPPORTED_EXTENSION for MOZ_hubs_components. I just tested a uv scroll component and it seems to work but also gave me that warning.

Other things to check: 
- the speed and increment properties
- your version of Blender(I just tested on 2.9, with  v0_0_4 of the hubs blender addon)
- how you set up your UV's

**Reply by cvanmeurs** [reply]
Can I see a screenshot of your material node setup?

**Reply by j-conrad** [reply]
> The gltf validator seems to always give us the warning of UNSUPPORTED_EXTENSION for MOZ_hubs_components.

This is correct. The glTF validator doesn't know about (most?) extensions so you'll see that warning sometimes. It's a bit misleading that there's a problem, but it's a normal warning you can usually ignore.
